src/test/java/uk/gov/companieshouse/officerfiling/api/controller/OfficerFilingControllerImplValidationIT.java

```diff
@@ -241,53 +241,6 @@ void createFilingWhenReferenceResignedOnInFutureThenResponse400() throws Excepti
                         is("must be a date in the past or in the present")));
```

Comment:
Does it even make sense to have an OfficerFilingControllerImplValidationIT any more? There is already an IT class for this controller, so if this class is specifically for validation then is it still required?

Reply:
Good point, but yes we still need it at the moment, as basic validation still needs to be run and there are checks in here that are not within the other IT class. like date not in future etc.
We can create a new non rush technical debt story and get the code merged into the main IT class from the validation IT class.